lib/order_repository.rb

```diff
@@ -0,0 +1,41 @@
+require '../lib/order'
+
+class OrderRepository
+  def initialize(io)
+    @io = io
+  end
+
+  def all
+    query = 'SELECT * FROM orders;'
+    result = DatabaseConnection.exec_params(query, [])
+    orders = []
+    result.each do |order|
+        new_order = Order.new
+        new_order.id = order['id']
+        new_order.order_date = order['order_date']
+        new_order.customer_name = order['customer_name']
+        new_order.item_id = order['item_id']
+        orders << new_order
+    end
+    return orders
+  end
+
+  def create
+    @io.puts "What is the customer name?"
```

Comment:
this method might be long and confusing. Maybe you could include some parts of this method in the app.rb